src/cljc/matcher_combinators/standalone.cljc

```diff
@@ -3,14 +3,37 @@
             [matcher-combinators.core :as core]
             [matcher-combinators.parser]))
 
+(defn match
+  "Returns a map indicating whether the `actual` value matches the `matcher`.
+
+  `matcher` can be a matcher-combinators matcher, a predicate function of
+  actual, an expression that returns a value, or a literal value.
+
+  `actual` can be an expression that returns the actual value, or a literal.
+
+  Return map includes the following keys:
+
+  - :matcher-combinators.result/type
+    - either :match or :mismatch
+  - :matcher-combinators.result/value
+    - when :matcher-combinators.result/type is :match, binds to the actual value,
+      else a map indicating the expected and actual values "
```

Comment:
It isn't always a map, it can be either a mismatch annotation or a composite datastructure where elements inside have mismatch annotations.

for example:
 - https://github.com/nubank/matcher-combinators/blob/574945ed6c982fe6381c8ffd80f6034dd607f68a/test/clj/matcher_combinators/core_test.clj#L162
 - https://github.com/nubank/matcher-combinators/blob/574945ed6c982fe6381c8ffd80f6034dd607f68a/test/clj/matcher_combinators/core_test.clj#L62